### ISSUE TITLE
Issue on page /global-patching.html

### ISSUE THREAD

**Issue by vpekar** [intel/scikit-learn-intelex#915]
Using the versions available via PyPI and Anaconda, trying to apply a global patch gives the error:
```
>>> from sklearnex import patch_sklearn
>>> patch_sklearn(global_patch=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: patch_sklearn() got an unexpected keyword argument 'global_patch'
```

**Comment by Pahandrovich** [comment]
Hi @vpekar, 
Which version of scikit-learn-intelex are you using?

**Comment by vpekar** [comment]
it's the one currently available via conda: 2021.20210714.120553
daal4py: 2021.3.0
scikit-learn: 0.24.2

**Comment by Pahandrovich** [comment]
Global patching is only supported from version 2021.4. You can update the version using one of three conda channels: intel, conda-forge, main or using PyPi.
`conda install -c intel scikit-learn-intelex`
`conda install -c conda-forge scikit-learn-intelex`
`conda install scikit-learn-intelex`
`pip install scikit-learn-intelex`
We will add information about which version this feature is supported from.
